### PR TITLE
feat: Add styling arguments to main sequence plot

### PR DESCRIPTION
**Pull request by theDebbister**
## Description

Fixes issue #449 

## Implemented changes

- [x] Added styling args to main sequence plot: color, marker_size, ...
- [x] possible to add further kwargs for plot.scatter

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change is or requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by SiQube** [comment]
the type for **kwargs comes from [plt scatter documentation](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.scatter.html)

File: src/pymovements/plotting/main_sequence_plot.py [hunk 1]
```diff
@@ -24,25 +24,39 @@
...
         event_df: EventDataFrame,
+        marker_size: float = 25,
+        color: str = 'purple',
+        alpha: float = 0.5,
+        marker: str = 'o',
         figsize: tuple[int, int] = (15, 5),
         title: str | None = None,
         savepath: str | None = None,
         show: bool = True,
+        **kwargs: Collection,
```
Comment (dkrako) [inline src/pymovements/plotting/main_sequence_plot.py:43]: I'm not sure if I understand the typehint here, but if mypy is happy, I am also happy

**Review by dkrako — COMMENTED** [review]
Can you also update the tutorial a bit with some of the new parameters?

That would be my only change request.
Apart from that, this looks fine!

**Review by dkrako — CHANGES_REQUESTED** [review]
.

**Review by dkrako — APPROVED** [review]
LGTM!

**Pull request merged**